gql/transport/transport.py

```diff
@@ -17,6 +20,24 @@ def execute(self, document: DocumentNode, *args, **kwargs) -> ExecutionResult:
             "Any Transport subclass must implement execute method"
         )  # pragma: no cover
 
+    @abc.abstractmethod
+    def execute_batch(
+        self,
+        reqs: List[GraphQLRequest],
+        *args,
+        **kwargs,
+    ) -> List[ExecutionResult]:
+        """Execute multiple GraphQL requests in batch.
+
+        Execute the provided requests for either a remote or local GraphQL Schema.
+
+        :param reqs: GraphQL requests as an iterable of GraphQLRequest objects.
+        :return: iterable of ExecutionResult
+        """
+        raise NotImplementedError(
+            "Any Transport subclass must implement execute_batch method"
```

Comment:
The text could be modified to something like: "This Transport has not implemented the execute_batch method"

Reply:
yes, thank you, I need to modify all the docstrings to match the new code